ci/eval/default.nix

```diff
@@ -152,8 +160,16 @@ let
         # Record and print stats on free memory and swap in the background
         (
           while true; do
-            availMemory=$(free -b | grep Mem | awk '{print $7}')
-            freeSwap=$(free -b | grep Swap | awk '{print $4}')
+            ${if stdenv.isDarwin then ''
+              freeBlocks=$(vm_stat | grep free | awk '{ print $3 }' | sed 's/\.//')
+              inactiveBlocks=$(vm_stat | grep inactive | awk '{ print $3 }' | sed 's/\.//')
+              speculativeBlocks=$(vm_stat | grep speculative | awk '{ print $3 }' | sed 's/\.//')
```

Comment:
Minor suggestion, we can do a single call to `vm_stat` and also do the grepping in awk:

```bash
output=$(vm_stat | sed 's/\.//g')
freeBlocks=$(awk '/free/ { print $3 }' <<<$output)
inactiveBlocks=$(awk '/inactive/ { print $3 }' <<<$output)
speculativeBlocks=$(awk '/speculative/ { print $3 }' <<<$output)
```